lib/passport-wsfed-saml2/saml.js

```diff
@@ -59,7 +59,7 @@ SAML.prototype.validateSignature = function (xml, options, callback) {
           var base64cer = embeddedSignature[0].firstChild.toString();
           var shasum = crypto.createHash('sha1');
           var der = new Buffer(base64cer, 'base64').toString('binary');
-          shasum.update(der);
+          shasum.update(der, 'latin1');

```

Comment:
We have `thumbprint` running this as well, it could be extracted.
Anyway, why don't simply passing the buffer to `shasum.update()` and we avoid 2 convertions, the first one to binary/latin1 and the other one (internally in `crypto` from latin1 to buffer.